pandas/io/tests/test_common.py

```diff
@@ -33,7 +33,8 @@ class TestCommonIOCapabilities(tm.TestCase):
 foo2,12,13,14,15
 bar2,12,13,14,15
 """
-
+    data2 = data1.replace(",", " ")
+    
     def test_expand_user(self):
```

Comment:
It's to use the regular expression and make sure I'm going through the code path I modified (I need a non default sep).
Also see my current issue with the file (not run by nosetests by default?).